From 2a540b1198c0ac84cfed14c5d1268193dde7aed9 Mon Sep 17 00:00:00 2001
From: Akshay Pradip Bharad <apbharad@habana.ai>
Date: Thu, 13 Jun 2024 15:50:01 +0300
Subject: [PATCH] [SW-180743] gaudi3: hbw drain timeout

This is an experimental patch to debug the cpupcp pkt read failure
issue. To be reverted after root cause is identified.

Change-Id: I7d8d6233410d0414b0c2e6db33f7a8d33959bf73
Signed-off-by: Akshay Pradip Bharad <apbharad@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index a8e1e48..5410deb 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -329,7 +329,10 @@
 	regmap_writel(PSOC_ARC2_PRELOADED_ID, mmD0_PARC_ARC2_CFG_BASE + mmARC_ARC_NUM);
 }
 
-#define HBW_DRAIN_TIMEOUT	0x8000
+/* This is an experimental change to debug host communication failure issue.
+To be reverted after root cause is identified */
+
+#define HBW_DRAIN_TIMEOUT	0xF0000
 #define LBW_DRAIN_TIMEOUT	0xF000
 
 void ppboot_pcie_drain_init(void)
